<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transactionWindow</class>
 <widget class="QDialog" name="transactionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transactions</string>
  </property>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>520</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Display Semib</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogout">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>520</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Display Semib</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>650</width>
     <height>400</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Display Semib</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_Graphic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Transaction">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>80</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_Graphic</zorder>
  <zorder>btnReturn</zorder>
  <zorder>btnLogout</zorder>
  <zorder>tableWidget</zorder>
  <zorder>label_Transaction</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
